test/wait-task/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
+                           Version 2.0, January 2004
```

Comment:
not sure we need the LICENSE file since the root of the repo already contains one

Reply:
Oh yeah, the two licenses are the same, removed.